include/zenoh-pico/net/query.h

```diff
@@ -33,7 +35,7 @@ typedef struct {
  * Return type when declaring a queryable.
  */
 typedef struct {
-    _z_zint_t _id;
+    uint32_t _id;
```

Comment:
Similar comment here, but I did not yet check the qid return size.

Reply:
This here is an entity id, they're u32